scripts/lua/rest/v1/get/contrib/nst/nst_network_load.lua

```diff
@@ -0,0 +1,72 @@
+--
```

Comment:
Endpoint is duplicate of `scripts/lua/rest/v1/get/interface/data.lua`. Merge with it.